README.md

```diff
@@ -0,0 +1,15 @@
+# talv
+
+Chess bot written in Rust. Comes with a simple GUI.
+
+## Download
```

Comment:
Maybe call it ## Download builds and also have a section for building it on your own

Reply:
sounds like a good idea! you should add that afterwards